README.md

```diff
@@ -56,6 +56,7 @@ If you're completly new to the world of P4 and are looking for step-by-step inst
 
 - [Installation guide: ONOS + Stratum + Barefoot Tofino switch](/stratum/docs/setup_guide_barefoot_tofino_onos.md)
 - [Trellis + Stratum example](/tools/mininet/examples/trellis/README.md)
+- [Continuous Certification Program (CCP)](https://certification.opennetworking.org/Stratum/)
```

Comment:
@bocon13 any better link? maybe the wiki link: https://wiki.opennetworking.org/display/COM/Stratum+Continuous+Certification+Program

Reply:
I think the wiki might be better.

Reply:
updated